cfme/tests/containers/test_reload_button_provider.py

```diff
@@ -20,22 +20,32 @@ def test_reload_button_provider(provider):
     """ This test verifies the data integrity of the fields in
         the Relationships table after clicking the "reload"
         button. Fields that are being verified as part of provider.validate.stats():
-        Projects, Routes, Container Services, Replicators, Pods, Containers, Nodes,
-        and Image Registries. Images are being verified separately, since the total
-        number of images in CFME 5.7 includes all images from the registry as well
-        as the images that are being created from the running pods
-
+        Projects, Routes, Container Services, Replicators, Pods, Containers, and Nodes.
+        Images are being verified separately, since the total
+        number of images in CFME 5.7 includes all images from the OSE registry as well
+        as the images that are being created from the running pods. The images are searched
+        according to the @sha.
     """
 
     navigate_to(provider, 'Details')
     tb.select('Reload Current Display')
     provider.validate_stats(ui=True)
 
-    num_img_from_registry = len(provider.mgmt.o_api.get('image')[1]['items'])
-    num_img_from_openshift = len(provider.mgmt.list_image())
+    l1 = provider.mgmt.list_image()
+
+    l1splitted = [i[2].split('@sha256:')[-1] for i in l1]
+
+    l2 = provider.mgmt.list_image_openshift()
+
+    l2splitted = [d['name'] for d in l2]
+
+    l2splitted_new = [i[7:].split('@sha256:')[-1] for i in l2splitted]
+
+    for s in l2splitted_new:
```

Comment:
This doesn't make much sense to me. What should it do?

Also, you are modifying `l2splitted_new`. I am not sure of current Python state but it used to be a no-no to do that while in for loop. This says it is not safe: https://docs.python.org/3/tutorial/controlflow.html#for-statements